src/main/java/com/foober/foober/model/Driver.java

```diff
@@ -0,0 +1,12 @@
+package com.foober.foober.model;
+
+import com.foober.foober.model.enumeration.DriverStatus;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class Driver extends User {
+    private DriverStatus status;
```

Comment:
Maybe it's better to call this `availability` to differentiate this from `DriveStatus`

Reply:
I think we should change `DriveStatus` to `RideStatus` which will resolve said problem.